graph/multi/undirected_test.go

```diff
@@ -24,9 +24,9 @@ func TestMaxID(t *testing.T) {
 		g.AddNode(i)
 		nodes[i] = struct{}{}
 	}
-	g.RemoveNode(Node(0))
+	g.RemoveNode(int64(0))
```

Comment:
I suspected you had a reason.

Reply:
Overall, these tests need to be rewritten since they are very ad hoc, but whenever I think of doing it, I can always find something else that I need to do more urgently.